analyses/fusion_filtering/run_fusion_merged.sh

```diff
@@ -27,39 +27,49 @@ results_path="results/"
 
 
 # fusion files before and after standardization
-arriba_file="${data_path}/pbta-fusion-arriba.tsv.gz"
-starfusion_file="${data_path}/pbta-fusion-starfusion.tsv.gz"
+arriba_file="${data_path}/fusion-arriba.tsv.gz"
+starfusion_file="${data_path}/fusion-starfusion.tsv.gz"
 standard_arriba_file="${scratch_path}/arriba.tsv"
 standard_starfusion_file="${scratch_path}/starfusion.tsv"
 
 # general filtering parameters
-artifact_filter="GTEx_Recurrent|DGD_PARALOGS|Normal|BodyMap"
+artifact_filter="GTEx|HGNC_GENEFAM|DGD_PARALOGS|Normal|BodyMap"
```

Comment:
I thought I saw @kgaonkar6 also comment on this, but I am not sure what the downstreaam implications of this change are - why is this changed?

Reply:
Yes. So if I leave the GTEx_recurrent there, it would throw an error message saying that "GTEx_Recurrent" does not exist or something - this filter used here is based on the sample script of 02. But I can use the original setup if you think this is more accurate.

Reply:
@kgaonkar6 has there been some update in FusionAnnotator that dropped this?

Also concerned about `HGNC_GENEFAM` - what does adding this do?

Reply:
Per our [annoFuse paper](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-020-03922-7), 

```Of note, we decided not to remove genes annotated in HGNC_GENEFAM, as this database contains multiple oncogenes and their removal resulted in missed true fusions using our validation truth set. ```

So, I do not think we want to add this filter now.

Reply:
There has been no update in the FusionAnnotator used, we just don't have the `GTEx_Recurrent` and `GTEx` as you will see from searching `GTEx_Reccurrent` in page here: https://github.com/FusionAnnotator/FusionAnnotator/wiki/Home/3953b06bebecb0703acb823ae562c8f11fe69ce4

So only if it has `GTEx_Reccurrent` then we remove it.

Since the PR is only related to script `00` this specific issue is not discussed here even though bash script was updated  for the overall script from the original PR which had all the updates.

Reply:
This change is discussed part of the PR 2 here https://github.com/PediatricOpenTargets/OpenPedCan-analysis/pull/37#discussion_r664585166

Reply:
```suggestion
artifact_filter="GTEx|DGD_PARALOGS|Normal|BodyMap"
```

Reply:
Hi @runjin326  sorry @jharenza discussed it further, there are some fusion annotated as `GTEx` that are also seen in cancer databases so we don't want to remove them here. 

 Can you keep the artifact filter as it was from the original script. That we are consistent with previous projects `GTEx_Recurrent|DGD_PARALOGS|Normal|BodyMap`.


Also, just confirming, you get this warning not an error right?
```
Warning message:
In fusion_filtering_QC(standardFusioncalls = standardFusioncalls,  :
  No fusion calls with annotation: GTEx_Recurrent
```

Reply:
Correct - this is what I got - just to confirm, we keep it as GTEx_Recurrent not GTEx and it's ok to get the warning correct?

Reply:
Correct, we have the warning coded to just let the user know that we didn't find `fusion calls with annotation: GTEx_Recurrent`.